docs/guides/postgres/enterprise/postgres-enterprise.md

```
@@ -0,0 +1,563 @@
+```bash
+$ kubectl get pg -n demo pg-enterprise
+NAME            VERSION         STATUS   AGE
+pg-enterprise   16.9-appscode   Ready    3h
+```
+
+If you want positive confirmation of *which* certificate is in force — after a
+renewal window closes, say — restart the pod and read the fresh startup line,
+which reports the serial and fingerprint of the file as it is now:
+
+```bash
+$ kubectl delete pod -n demo pg-enterprise-0
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)